spec/menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'menu'
+
+describe Menu do 
+  
+  it { is_expected.to respond_to (:display_menu) }
+
+  it "displays the list of dishes and their prices" do
+    menu = Menu.new
+    expect{ subject.display_menu }.to output(@list).to_stdout
```

Comment:
Great use of 'output' and 'to_stdout' here. Great readability and it coincides nicely with the code you have written 😄.